### DISCUSSION TITLE
Pulumi SaaS?

### DISCUSSION THREAD

**Discussion by prologic** [pulumi/pulumi#10024] [General]
Hello all 👋

This question has probably been asked before, but I just recently came across Pulumi (_funnily enough from interviewing a candidate for my SRE team_).

Quick question:

Looking at the [Pulumi landing page](https://www.pulumi.com/) I happened to notice this [Pricing page](https://www.pulumi.com/pricing/).

Q: Can Pulumi be used without this?

What is this SaaS thing anyway?

Kind regards

James

**Comment by mattstratton** [comment]
The Pulumi SaaS is one of several options for the state backend. You don't have to use it, but it's the easiest method. 

More info: https://www.pulumi.com/docs/intro/concepts/state/

**Reply by prologic** [reply]
Thanks for your quick response! This makes perfect sense, as Terraform has a similar concept.

The reason I'm asking about this, is we have a strong requirement that means we either have to self-host the Pulumi backend for state or use some other state backend. For example we have S3/Object-Storage we _could_ use. I'll check out that doc and have a read...

Thanks! 🙏

**Comment by prologic** [comment]
Oh one more final question... (_havenlt found this yet..._)

is the Pulumi backend free to use / stand-up? Or is that Enterprise / on-preom only?

I was looking at the [Self hosting option](https://www.pulumi.com/docs/intro/concepts/state/#logging-into-a-self-hosted-pulumi-service-backend) 👈 here...

**Reply by prologic** [reply]
if so that would be really cool 😎

**Reply by susanev** [reply]
the open source backends are free to use. self-hosting the pulumi service requires a subscription.

**Reply by prologic** [reply]
I see. Thanks!

**Comment by susanev** [comment]
@prologic if you have thoughts on how we can make this information clearer on our pricing page or in the docs, pls let us know, thank you!!

**Reply by prologic** [reply]
I'm still reading, I'm quite time poor so I asked here. Hope that's okay 🙇‍♂️

**Reply by susanev** [reply]
Oh yes, of course!! Asking here is perfect, but also want to make this information more accessible for everyone so we welcome ideas if ya have them!

**Reply by begerard** [reply]
If I may, I just discovered the FOSS part of your solution, always though we could only use pulumi with pulumi service (and only did thanks to an Azure blog article listing you as a FOSS alternative). I found my way in your documentation about self hosting thanks to your link in the pricing page ("Prefer to manage your own state storage backend?"), but I tried this because I know you have it hiding somethere!

It may not be wize commercially, but for a better discoverability it would help to have it as a column below the link, along your other offers. Of course you would need to clearly separate it (with different background for example), but unfortunately we all have the bad habit of jumping on the most graphically presented elements and not reading the few lines over them...
